src/main/webapp/index.jsp

```diff
@@ -286,7 +286,7 @@ h2.vidcap {
 	                           Encounter thisEnc=latestIndividuals.get(i);
 	                           %>
 	                            <li>
-	                                <img src="images/wild_dog_logo_2.png" alt="" width="85px" height="75px" class="pull-left" />
+	                                <img src="images/WNWB_wolf_lowres.png" alt="" width="85px" height="75px" class="pull-left" />
```

Comment:
think this `WNWB_wolf_lowres.png` image also needs to be added to this PR as well.